docs/low-code-spark/fabrics/synapsefabric.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Azure Synapse Fabric"
+id: azure-synapse-fabric-guide
+description: fabric configuration guide
+sidebar_position: 4
+tags:
+  - deployment
+  - configuration
+  - azure
+  - synapse
+  - livy
+---
+
+In the context of Spark execution engines, users have the flexibility to opt for Azure Synapse Analytics while utilizing Prophecy. This comprehensive documentation aims to provide users with a clear understanding of the configuration process for Azure Synapse Analytics and Apache Livy. Additionally, it offers step-by-step guidance on creating a Fabric that enables seamless connectivity to the Azure Synapse environment via Livy.
+
+Feel free to explore the following sections to gain insights into the integration of Azure Synapse Analytics and Apache Livy with Prophecy.
```

Comment:
No need to mention Livy here. I don't think users are interested in the internals of how we connect to their Azure Synapse workspace

Reply:
Yeah, in the context of Synapse, "Livy" is just an implementation detail Internal to Azure.